sky/shell/platform/ios/FlutterViewController.mm

```diff
@@ -183,17 +187,27 @@ - (void)setupPlatformServiceProvider {
   mojo::ServiceProviderPtr viewServiceProvider;
   new sky::shell::ViewServiceProvider(mojo::GetProxy(&viewServiceProvider));
 
+  mojo::ServiceProviderPtr outgoingServiceProvider;
+
   sky::ServicesDataPtr services = sky::ServicesData::New();
   services->incoming_services = serviceProvider.Pass();
+  services->outgoing_services = mojo::GetProxy(&outgoingServiceProvider);
   services->view_services = viewServiceProvider.Pass();
   _engine->SetServices(services.Pass());
+
+  semantics::SemanticsServerPtr semantics_server;
+  mojo::ConnectToService(outgoingServiceProvider.get(), &semantics_server);
+  _semanticsServer = semantics_server.Pass();
 }
 
 #pragma mark - Loading the view
 
 - (void)loadView {
   FlutterView* surface = [[FlutterView alloc] init];
 
+  // Construct the listener implementation
+  new sky::shell::SemanticsListenerImpl(surface, _semanticsServer.Pass());
```

Comment:
We only create them here, and we do so right after we create the view so we can hand the view reference to the listener.

loadView() will only be called once -- it's called by the controller automatically when the view property is nil to lazily load the view.

It's retained for the life of the process, much in the same way the platform service provider and view service provider are.  I suppose we could keep references to these objects and delete them in dealloc, but it doesn't seem worth it since the controller is a singleton?